### ISSUE TITLE
L1R1 and L2R2 buttons rework

### ISSUE THREAD

**Issue by marcos-diaz** [inputlabs/organization#95]
L1R1 buttons should be easier to press without having to move the finger up so much.

One obvious solution would be making L1R1 buttons to be taller (extending down), but 3D-printing may require supports, which is not ideal.
